lambda/custom/index.js

```diff
@@ -15,9 +15,31 @@ const APL_DOC = require ('./document/renderPage.json' ) ;
 const TWO_PAGER_COMMANDS =  require ('./document/twoSpeakItemsCommand.json' ) ;
 const ONE_PAGER_COMMANDS =  require ('./document/oneSpeakItemCommand.json' ) ;
 const TOKEN_ID = 'pagerSample';
-let MAIN_DATA_SOURCE = require ('./document/dataSource.json' );
-
-
+let firstTransformer = [
+      {
+          "inputPath": "phraseSI",
+          "outputName": "phraseAsSpeech",
+          "transformer": "ssmlToSpeech"
+      }
+    ];
+let secondTransformer = [
+      {
+          "inputPath": "phraseSI",
```

Comment:
What does phraseSI Mean? Can you make this clearer?

Reply:
Done